lwallet/Main.hs

```diff
@@ -315,8 +325,11 @@ runCmd _ ListAddresses _ = do
    putText "Available addresses:"
    for_ (zip [0 :: Int ..] addrs) $ \(i, pk) -> do
        addr <- makePubKeyAddressLWallet pk
-       putText $ sformat ("    #"%int%":   "%build%" (PK: "%stext%")")
-                    i addr (toBase58Text pk)
+       putText $ sformat ("    #"%int%":   addr:      "%build%"\n"%
+                          "          pk base58: "%stext%"\n"%
+                          "          pk hex:    "%fullPublicKeyHexF%"\n"%
+                          "          pk hash:   "%hashHexF)
```

Comment:
Would be good to also write that it's called stakeholder id (`pk hash (stakeholder id): bla`).

Reply:
At first I did it, but it breaks the alignment.